internal/pkg/template/input/step.go

```diff
@@ -0,0 +1,18 @@
+package input
+
+type StepsGroups []StepsGroup
+
+type StepsGroup struct {
+	Description string `json:"description" validate:"max=200"`
+	Required    string `json:"required" validate:"oneof=all atLeastOne exactOne zeroOrOne optional"`
+	Steps       []Step `json:"steps" validate:"required,dive"`
```

Comment:
Potrebujeme aspon jeden step v group, `required` osetri `nil` hodnotu ale nie prazdnu `slice`.
Teda namiesto `required` by som dal `min=1`.

Je to taka specificka vec pre Go, ze `slice` moze byt nealokovana -> `nil`, alebo prazdna.
Vacsinou su tieto stavy rovnocenne (napr. `for := slice`) , ale tu nie.

Pozri: https://go.dev/play/p/gTnMuydKAYE